### PR TITLE
Labels

### PR DESCRIPTION
**Pull request by whitmo**
This is a WIP (DO NOT MERGE) and you will note several things undone:

 - requires customizations to charmhelpers ([to point ansible-playbook at the custom module](https://github.com/whitmo/charmhelpers/commit/ba32dd9bdea7e921e999aff8588df27e3dae3195))
 - need to cleanup hooks.py & setup.py
 - need to actually use module to set and gen opts

But the basics work and hopefully will provide a generally clean way of managing the options.  I could  extend the module to be more of a general purpose data manager, but this works for what we need.

### CODE REVIEW

**Comment by whitmo** [comment]
meh. first and last commit are only ones that matter.

File: hooks/hooks.py [hunk 1]
```diff
@@ -1,28 +1,14 @@
 #!/usr/bin/env python
-from charmhelpers.contrib.ansible import AnsibleHooks
-from charmhelpers.contrib.ansible import install_ansible_support
+from ansiblecharm.runner import AnsibleHooks
```
Comment (chuckbutler) [inline hooks/hooks.py]: Breakout from charm helpers, ok. I can see where this is going. this is our pretext to maintaining a module that relates to but doesn't live in charm helpers right?

File: hooks/python-pkgs.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-e git+https://github.com/whitmo/charmhelpers.git#egg=charmhelpers
```
Comment (chuckbutler) [inline hooks/python-pkgs.txt]: These are scary to me, I dont look at these files often. Do we have open PR's to get these into upstream?

File: metadata.yaml [hunk 3]
```diff
@@ -10,6 +10,8 @@ subordinate: false
 provides:
   docker-containers:
     interface: containers
+  events:
```
Comment (chuckbutler) [inline metadata.yaml]: +1 to the new interface, we need to get docs around what its sending added to the readme/docsite.

**Pull request merged**